<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify_username_dia</class>
 <widget class="QDialog" name="Modify_username_dia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改用户名</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	
	background-image:url(:/new/prefix1/picture/back.jpg)
}</string>
  </property>
  <widget class="QPushButton" name="modify_enter_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:rgb(166,166,166);
color:white;
border:1px solid rgb(168,168,168);
background:rgba(0,0,0,0);
}

QPushButton:hover
{
background-color:rgba(239,239,239,1);
color:back;
background:rgba(166,166,166,1);
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modify_nsername_edit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>346</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border:1px solid balck;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="modify_nsername_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:white; 
}
</string>
   </property>
   <property name="text">
    <string>新用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modify_cancle_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:rgb(166,166,166);
color:white;
border:1px solid rgb(168,168,168);
background:rgba(0,0,0,0);
}

QPushButton:hover
{
background-color:rgba(239,239,239,1);
color:back;
background:rgba(166,166,166,1);
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="modify_tooltip_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>471</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:white; 
}
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aaff;&quot;&gt;请按正确的格式输入，允许使用中文、字母、数字&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
